services/platform/docker-entrypoint.sh

```
@@ -119,6 +119,22 @@ if [ -z "${RAG_DATABASE_URL:-}" ] && [ -n "${POSTGRES_URL:-}" ]; then
 
+# Admin key + WebDAV HMAC key. Both must be readable from `bun server.ts`'s
+# process.env, not just the deploy_convex_functions function. `ADMIN_KEY`
+# was previously a `local` and silently went out of scope before the
+# platform Hono server started — every `/dav/*` request then 500'd. Both
+# are deterministic from $INSTANCE_SECRET so they survive restarts and
+# operators don't need to set them by hand.
+#
+# `WEBDAV_APP_PASSWORD_HMAC_KEY` accepts an explicit override (.env) for
+# operators who want a key rotation independent of INSTANCE_SECRET. When
+# unset, we derive a 64-char hex via sha256(secret || ':webdav-hmac:v1').
+export ADMIN_KEY="$(generate_key "$INSTANCE_NAME" "$INSTANCE_SECRET")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)